build_scripts/windows/scripts/az

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 
-AZ_INSTALLER=MSI "$(dirname "${BASH_SOURCE[0]}")/../python.exe" -IBm azure.cli "$@"
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+export PYTHONPATH="$(dirname "$SCRIPT_DIR")"
+AZ_INSTALLER=MSI "$SCRIPT_DIR/../python.exe" -IBm azure.cli "$@"
```

Comment:
The root cause of the crash is Python 3.12.8 loads 3.12.x's module(`C:\\Users\\xxx\\AppData\\Local\\Programs\\Python\\Python312\\Lib`) when run `pip`.(https://github.com/Azure/azure-cli/issues/30594#issuecomment-2646862107)

We have `-I` option when run `az`, but we don't have it in `pip`.
https://github.com/Azure/azure-cli/blob/663212038fbbb42bda7498e426fce309f0c8f9df/src/azure-cli-core/azure/cli/core/extension/operations.py#L49

Even though we can add `-I` in Line 49, it may not take effect, see https://github.com/pypa/pip/issues/13023. Prepend `../` to `PYTHONPATH` seems to be the only workaround.